cli/command/swarm/testdata/update-noargs.golden

```diff
@@ -8,6 +8,7 @@ Flags:
       --cert-expiry duration            Validity period for node certificates (ns|us|ms|s|m|h) (default 2160h0m0s)
       --dispatcher-heartbeat duration   Dispatcher heartbeat period (ns|us|ms|s|m|h) (default 5s)
       --external-ca external-ca         Specifications of one or more certificate signing endpoints
+  -h, --help                            help for update
```

Comment:
ah, yes, think we previously special-cased `-h / `--help` and hid it, because using that flag will print the same information as is being printed here.

Reply:
Ya, I think we might have been relying on a side-effect of an issue that was fixed in cobra to hide this flag, after we migrated to cobra.  I think we can restore the old behaviour by hiding this flag. I'll try it out.